contracts/Vault.vy

```diff
@@ -1124,14 +1128,14 @@ def creditAvailable(_strategy: address = msg.sender) -> uint256:
 def _expectedReturn(_strategy: address) -> uint256:
```

Comment:
if totalGain does not include losses then you need to update this method. See my comment further down where you calculate loss in report() method